NeerajChaudhary.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "Neeraj Chadhary",
+  "skills": ['C++''Java'],
```

Comment:
Also, please add a comma between "c++" and "java" :)

Reply:
Yes it helped me